src/main/java/org/folio/inventory/support/CompletableFutures.java

```diff
@@ -0,0 +1,16 @@
+package org.folio.inventory.support;
+
+import java.util.concurrent.CompletableFuture;
+
+public final class CompletableFutures {
```

Comment:
This should be marked as deprecated with a comment saying "When we switch to Java 11 remove this class and use CompletableFuture#failedFuture(Throwable) instead. https://stackoverflow.com/a/49432327"

Reply:
@julianladisch When might FOLIO be upgrading to Java 11?

I've put deprecation warnings in other parts of the system, and they mostly generate noise if there isn't intent to replace the behaviour fairly quickly.

Reply:
I've added a comment to the method. I think Sonar will report an error if I explicitly add `@Deprecated` annotation.